### ISSUE TITLE
Positioning

### ISSUE THREAD

**Issue by xavisavvy** [shawnchin/jquery-gentleSelect#7]
Just curious if there is a way to change the positioning of the element from relative to the dropdown to say a centered on the window? I am having trouble with it going off the screen on mobile devices and was hoping to mod it or find an option to position it relative to the window.

**Comment by shawnchin** [comment]
I don't recall there being an official way to do that, however I believe [the css values that control that are set on `.data("dialog")`](https://github.com/shawnchin/jquery-gentleSelect/blob/c8eabf228cc5e242654d19ad4de466e2dc4d5cfe/gentleSelect/jquery-gentleSelect.js#L274) so I suppose you could target that element and change the position from outside the plugin. 

If you're changing the position of the dialog, you'll also want to set `hideOnMouseOut = false` or the dialog will hide itself if the position of the mouse does not overlap with it.

Apologies that I cannot take a more active role in helping you get this working. Other responsibilities are currently keeping my away from working on this project.

**Comment by xavisavvy** [comment]
No worries, I ended up doing similar to what you mentioned. It would be a great addition to your stuff if you add it in the future :) Really for me on smaller screens I just have it centering on the document that way there is no chance of it running off the screen with larger fields like seconds. Thank you for the reply!

**Comment by shawnchin** [comment]
Yes, that's certainly something worth looking at. I'll keep this issue open as a reminder :)
Many thanks for posting this.
